src/pybind/mgr/cephadm/ssl_cert_utils.py

```diff
@@ -37,7 +34,7 @@ def generate_root_cert(self) -> Tuple[str, str]:
         root_builder = root_builder.public_key(root_public_key)
         root_builder = root_builder.add_extension(
             x509.SubjectAlternativeName(
-                [x509.IPAddress(ipaddress.IPv4Address(str(self.mgr.get_mgr_ip())))]
+                [x509.IPAddress(ipaddress.IPv4Address(host))]
```

Comment:
> the root cert does need the ip to always be that of the active mgr I believe. Are we worried this change risks breaking that in the future by just allowing any call to this to pass an arbitrary host (or ip really I guess looking at how this is used). Should we at least make the arg name `mgr_ip` or something like that?

I moved the code to the `ssl_cert_utils.py` to make it generic so it could be used anywhere. Thus, I avoided to refer to the mgr in the code and in the args as well. It would be the user responsibility to pass the active mgr ip. Maybe we can add some comment there to make it clear.